### DISCUSSION TITLE
TAD instead of time

### DISCUSSION THREAD

**Discussion by Martin-Umpierrez** [FelicienLL/mapbayr#217] [Q&A]
Hello, first of all, thank you very much for this amazing package. I think it's extremely useful for those of us who work on TDM. I've been experiencing some troubles with steady state/time, and perhaps someone can help me out. Here is a brief excerpt from my dataset.

![image](https://github.com/FelicienLL/mapbayr/assets/140851573/9ac033be-0983-4dcd-b351-6d9785305a83)

I've encountered some difficulties when trying to reproduce what I'm looking for. The dataset I'm attaching is for a patient with three occasions. I would like to know how I can modify this dataset so that in the IPRED plot returned by the `mapbayr_plot()` function, I can obtain a continuous trace of the administered doses. Alternatively, I would like to understand why, in the case of time 264, the curve starts again from zero when I specified that it is an event in steady state, with a dose administered every 12 hours.

![image](https://github.com/FelicienLL/mapbayr/assets/140851573/61fcf98f-f477-4975-b3cb-fe6f9903768a)

Since I couldn't achieve what I wanted, I tried using TAD to bring everything to the same time-scale, as these are trough observations and I'm really interested in understanding how the predictions are behaving, but I was unsuccessful. It tells me that a TIME column is mandatory.

PS: I tried both with EVID 4 and 1 on Occasions 2 and 3.
This is a database adapted from the Monolix software, which works relatively differently from NONMEME databases, and perhaps there's something I'm misunderstanding.
I would appreciate any help someone can provide me

Regards, martin

**Comment by FelicienLL** [comment]
Dear Martin, 
I am not sure I understand what figure exactly you are expecting. If you want multiple doses to appear on the plot, they must be in the data set. One per row, or you can also use ADDL. Using SS will only consider that the administration with SS = 1 is the last of an infinity of identical administrations given every 12h, but they are not "really" in the data set.
I don't have straightforward solution for displaying time after dose on the X axis, it would depend on your data and model. But mrgsolve, as NONMEM, requires TIME. 
Hope it helps
Best regards
